src/schema/v2/partner/partnerCollectorProfile.ts

```diff
@@ -0,0 +1,37 @@
+import { GraphQLObjectType } from "graphql"
+import { CollectorProfileType } from "schema/v2/CollectorProfile/collectorProfile"
+import { ResolverContext } from "types/graphql"
+
+export const PartnerCollectorProfileType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "PartnerCollectorProfile",
+  fields: () => ({
+    collectorProfile: {
+      type: CollectorProfileType,
+      resolve: async (
+        { partnerId, collectorId },
+        _args,
+        { partnerCollectorProfileLoader }
+      ) => {
+        if (!partnerCollectorProfileLoader) return
+
+        try {
+          const data = await partnerCollectorProfileLoader({
+            partnerId,
+            userId: collectorId,
+          })
+
+          return {
+            ...data.collector_profile,
+            follows_profile: data.follows_profile,
```

Comment:
NIce! Not seeing where `follows_profile` (can be camel-cased here) is being used within this type?